.github/workflows/IKVM.yml

```diff
@@ -15,20 +15,20 @@ on:
 jobs:
   build-openjdk:
     name: Build OpenJDK
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
     - name: Checkout Source
-      uses: actions/checkout@v1
+      uses: actions/checkout@v3
       with:
         submodules: recursive
     - name: Cache OpenJDK Build
       uses: actions/cache@v3
       with:
         path: openjdk/build/linux-x86_64-normal-server-release
-        key: openjdk-build-linux-x86_64-normal-server-release--${{ runner.os }}--${{ hashFiles('openjdk/**', '!openjdk/build') }}-8
```

Comment:
All you need to do is put -9 on the end of it, not change the prefix.

Reply:
Done. Thank you!